drivers/hmis/app/graphql/types/hmis_schema/household.rb

```diff
@@ -54,16 +57,20 @@ def assessments(**args)
       resolve_assessments(**args)
     end
 
+    def current_staff_assignments
+      load_ar_association(object, :staff_assignments, scope: Hmis::StaffAssignment.order(created_at: :desc, id: :desc))
+    end
+
+    # This field results in N+1 because it is paginated. It is used for displaying the staff assignment history
+    # for a particular household. When loading staff assignments on a batch of households,
+    # use the 'current_staff_assignments' field instead.
     def staff_assignments(is_currently_assigned: true)
-      # There's no current use case for returning all (both currently assigned and formerly assigned)
```

Comment:
If I'm understanding correctly, this comment "There's no current use case for returning all (both currently assigned and formerly assigned) in the same query" is still relevant, right? That's the reason for the `is_currently_assigned` param

Reply:
We only ever call this with `isCurrentlyAssigned: false` now. I'll update the comment!